_posts/2020-07-17-maven-wrapper.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Make Your Build Environment-Independent with the Maven Wrapper
+categories: [java]
+date: 2020-07-17 05:00:00 +1100
+modified: 2020-07-17 05:00:00 +1100
+author: saajan
+excerpt: "A short article about Maven Wrapper - what problem it solves, how to set it up, and how it works"
+image:
+  auto: 0073-broken
+---
+
+In this article, we'll learn about the [Maven Wrapper](https://github.com/takari/maven-wrapper) - what problem it solves, how to set it up, and how it works.
+
+## What is the Maven Wrapper?
+
+Years ago, I was on a team developing a desktop-based Java application. We wanted to share our artifact with a couple of business users in the field to get some feedback. It was unlikely they had Java installed. Asking them to download, install, and configure version 1.2 of Java (yes, this was that long ago!) to run our application would have been a hassle for them. 
```

Comment:
@thombergs I found the similarity between the ideas here interesting and thought of sharing. If you think it's not necessary, let me know.

Reply:
No, I like it :)